### ISSUE TITLE
Need help with syntax in blink girl example

### ISSUE THREAD

**Issue by vcl00e** [FizzleDorf/ComfyUI_FizzNodes#78]
Hello! Thankyou for providing this project it is great mind-blowing stuff for me :) 

For the blink girl example, there is some syntax I couldn't find an explanation for in the docs. Can you please help elaborate a bit?

"0": "(:o:`pw_a`)",
"23": "(:o:0)", 
"35": "(:D:1), spoken heart",
"48": "(:D:0.6), spoken heart",
"71": "(:o:`pw_a`)",

The docs says this:
"#":"(prompt:`exp`)"

So for the first line, the prompt is :o  but I couldn't find out what :o meant.
Also, same for :D and the spoken heart after the comma.

Link to the doc I was reading:
https://github.com/FizzleDorf/ComfyUI_FizzNodes/wiki/Prompt-Schedules#prompt-syntax

Many thanks!
Vincent

**Comment by FizzleDorf** [comment]
`exp` in the wiki refers to the expression you want to use. expressions can be sine functions like I have later in the wiki section. the `pw_a` and others are variables you can control with a value schedule. anything available in numexpr can be evaluated inside the backticks. you can find the available functions for expressions in the documentation [here](https://numexpr.readthedocs.io/en/latest/user_guide.html#supported-operators) 

as an abstracted example it would be structured like:
```(prompt:`expression goes here`)```

a practical example would look something like:
```
"0": "(smiling:`(0.5+0.5*sin(t/12))`)",
"24": "(dancing:`pw_a`)",
"48": "(dancing:`pw_a`)",
"72": "(smiling:`(0.5+0.5*sin(t/max_f))`)"```
```

**Comment by vcl00e** [comment]
Hi FizzleDorf thank you for helping me! :) I understand the things in the documentation however..

What I meant to say is that there are things in the examples that are not explained in the documentation. 

"#":"(prompt:`exp`)" does not cover all the cases. For example the image from [Prompt Schedule Nodes docs](https://github.com/FizzleDorf/ComfyUI_FizzNodes/wiki/Prompt-Schedules) shows a new syntax that allows chaining of multiple prompt:exp in one line, and also adding prompt without expression. It looks like this:   "#":"(prompt:`exp`),(prompt:`exp`),prompt" In the example there is also a line that starts with "72": ":3,   and looks like there is an expression but no prompt. I do not understand what that means.

![image](https://github.com/FizzleDorf/ComfyUI_FizzNodes/assets/16975050/420d197a-8c6b-42bc-80e9-c629db558609)

Back to my original question. The workflow image I got from the [the main FizzNodes git page](https://github.com/FizzleDorf/ComfyUI_FizzNodes) works fine, but it also contains some mysterious prompt syntax I don't understand.

![image](https://github.com/FizzleDorf/ComfyUI_FizzNodes/assets/16975050/86e12e76-7175-462c-8688-c8846423a670)

I highlighted the mysterious prompt syntax in orange. It looks to be like some kind of variable prompt :o and ;D but I can't find what they are referring to. I hope I'm not being silly and miss something important :D  But I hope this helps maybe we can add some explanation to the doc to clarify if I'm not being silly!

Many thanks!
